src/courses/delta/03.md

```diff
@@ -0,0 +1,87 @@
+
+The process of preparing controls from one baseline to another (baseline X to Y) consists of:
+
+1. Providing the directory where baseline X controls are located.
+2. Providing or generating an InSpec JSON formatted file containing all baseline X controls.
+    - The JSON file is generated using the [InSpec or Cinc-Auditor] JSON CLI command.
+3. Providing an XCCDF file containing the new baseline Y guidance. The file is obtained from the appropriate site.
+    - The baseline Y XCCDF lists all controls appropriate for the provided baseline. It often includes a field (legacy) indicating the control's previous name(s). The process uses this field to map old control names to the new controls.
+    - If no mapping is found, then the `delta` process must be invoked to convert the baselines ([see use cases](./04.html#use-cases-for-running-delta)).
+
+You can invoke the command as follows:
+
+```sh
+saf generate update_controls4delta -X ./baseline_Y_xccdf_guidance_file.xml -J baseline_X_summary.json -c baseline_X_controls_directory
+```
+
+Additional capabilities can be obtained using the `-h or --help` flag on the `update_controls4delta` command as such:
+
+```sh
+saf generate update_controls4delta --help
+```
+
+Renaming of controls can be done either by guidance rules (rule ID, default) or using the group ID. When using rule ID legacy fields contained within the XCCDF are used for identifying the proper mapping for the controls.
+
+If there is no way to trace controls in baseline X to its corresponding controls in baseline Y, then `update_controls4delta` will not update any controls (no mapping).
+
+### 3.3 Delta
+
+The `delta` process extends the capabilities of the `update_controls4delta` process by providing the following functionalities:
+
+1. Creates new controls found in updated guidances
+2. Fuzzy matching capability (optional)
+    a - Maps controls based on similarity and not control IDs
+    b - For controls which a match is found, the describe block (code) within the old control is mapped over to the new control
+3. Detailed logging
+    a - report file (.md), mapping statistics (CliProcessOutput.log)
+
+You can invoke the command as such:
+
+```sh
+saf generate delta -X ./baseline_Y_xccdf_guidance_file.xml -J baseline_X_summary.json -o new_baseline_Y_controls_directory -M -c baseline_X_controls_directory
+```
+
+::: tip Optional Fuzzy Matching Flags
+The `-M` and `-c` flags can be removed if not using fuzzy matching feature.
+:::
+
+Additional capabilities can be obtained using the `-h or --help` flag on the `delta` command as such:
+
+```sh
+saf generate delta --help
+```
+
+For more information on these commands, refer to the following documentation:
+
+- [update_controls4delta](https://saf-cli.mitre.org/#delta-supporting-options)
+- [saf generate delta](https://saf-cli.mitre.org/#delta)
+
+::: warning Delta Process Limitations
+The Delta process has been tested using DISA XCCDF STIG Benchmark Guidances only.
```

Comment:
Why not get rid of this warning and only advertise it for DISA XCCDF STIGs?

Reply:
I agree it needs further testing before advertising it